include/cutlass/epilogue/dispatch_policy.hpp

```diff
@@ -131,13 +131,15 @@ template<
   int StagesC_,
   int StagesD_,
   int FragmentSize_,
-  bool ReuseSmemC_
+  bool ReuseSmemC_,
+  bool EnableNullptrD_ = true
```

Comment:
Nit: should the `EnableNullptrD_` be `false` by default?

Reply:
Updated

Reply:
Thank you. Cleared on my side.